packages/rich-text/src/component/event-listeners/prevent-focus-capture.js

```diff
@@ -0,0 +1,35 @@
+export default () => ( element ) => {
+	const { ownerDocument } = element;
+	const { defaultView } = ownerDocument;
+
+	let value = null;
+
+	function onPointerDown( event ) {
+		// Abort if the event is default prevented, we will not get a pointer up event.
+		if ( event.defaultPrevented ) {
+			return;
+		}
+		if ( event.target === element ) {
+			return;
+		}
+		if ( ! event.target.contains( element ) ) {
+			return;
+		}
+		value = element.getAttribute( 'contenteditable' );
+		element.setAttribute( 'contenteditable', 'false' );
+	}
+
+	function onPointerUp() {
+		if ( value !== null ) {
+			element.setAttribute( 'contenteditable', value );
+			value = null;
+		}
+	}
+
+	defaultView.addEventListener( 'pointerdown', onPointerDown );
+	defaultView.addEventListener( 'pointerup', onPointerUp );
```

Comment:
To do: we actually don't need to add this listener to every instance, only once would be enough.

Reply:
But you'd need a way to keep separate `value` state for each instance, right?

Reply:
I guess, unless we set on the element temporarily.